package.json

```
@@ -37,6 +38,7 @@
     "eslint-plugin-mocha": "^10.0.5",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-unicorn": "^45.0.0",
+    "eslint-remote-tester": "^3.0.0",
```

> Looks like there's a CI failure related to this: [AriPerkkio/eslint-remote-tester#382](https://github.com/AriPerkkio/eslint-remote-tester/issues/382)

The package has to be linked, exactly as done here: https://github.com/eslint-community/eslint-plugin-eslint-plugin/blob/3ffd85ed3957d907e7386af9d4e4e471ccf671c6/package.json#L62.


```suggestion
    "eslint-plugin-node": "^11.1.0",
    "eslint-plugin-qunit": "file:./",
    "eslint-plugin-unicorn": "^45.0.0",
    "eslint-remote-tester": "^3.0.0",
```

---

Fixed, thanks.